custom_components/opensprinkler/switch.py

```diff
@@ -71,13 +71,21 @@ def __init__(self, entry_id, name, controller, coordinator):
     @property
     def name(self):
         """Return the name of controller switch."""
-        return f"{self._name} Operation Enabled"
+        return f"{self._name} Enabled"
 
     @property
     def unique_id(self) -> str:
         """Return a unique, Home Assistant friendly identifier for this entity."""
         return f"{self._entry_id}_{self._entity_type}_controller_operation_enabled"
 
+    @property
+    def icon(self) -> str:
+        """Return icon."""
+        if self._controller.operation_enabled:
+            return "mdi:circle"
```

Comment:
Leave as lightning

Reply:
@vinteo any interest in the `power-*` icons?

Reply:
how about something a bit more out there... `barley` and `barley-off`

Reply:
or `leak` and `leak-off`

Reply:
I like the `barley` better than the `leak` :D

Reply:
Let's change it barley and merge this